recipe/meta.yaml

```diff
@@ -12,34 +12,47 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  Skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
```

Comment:
`--no-build-isolation`, the second time I see this today...did we decide to make it part of the run-of-the-mill build command?

Reply:
it is part on the linter requriement, actually it was generated by the linter --fix command, is there anything I missing here ?

Reply:
It was decided more than 6 months ago. On this package that doesn't change much, but on packages with build depdencies set in pyproject.toml, that prevents pip from building in isolated mode, disregarding our deps and fetching from pypi.